sandboxed_containers/uninstalling-sandboxed-containers.adoc

```diff
@@ -0,0 +1,20 @@
+[id="uninstalling-sandboxed-containers"]
+= Uninstalling {sandboxed-containers-first}
+include::modules/common-attributes.adoc[]
+:context: uninstalling-sandboxed-containers
+
+toc::[]
+
+== Uninstalling OpenShift sandboxed containers using the web console
+You can uninstall {sandboxed-containers-first} by using the {product-title} web console.
+
+include::modules/sandboxed-containers-deleting-operator-deployment-cr.adoc[leveloffset=+2]
+include::modules/sandboxed-containers-deleting-namespace-web-console.adoc[leveloffset=+3]
+include::modules/sandboxed-containers-deleting-operator-web-console.adoc[leveloffset=+2]
```

Comment:
Blank lines between?

Just checking that you want the middle one at +3 for a reason.